### PR TITLE
Update templates.yml update Atom

### PR DESCRIPTION
**Pull request by AnielMCU**

### CODE REVIEW

File: Citizens/templates.yml [hunk 1]
```diff
@@ -8006,10 +8006,10 @@ atom:
         pitch: 0.0
     metadata:
       player-skin-textures: eyJ0aW1lc3RhbXAiOjE0NzI3ODAzMDkyMzUsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiZDczOTcxMTY2MTM2YjEyNmZmN2U1Y2VmYzIyNTgzODYxYTczZjU3NjFiMTBmNTg4OGUzZDg5ZGE1ZTllMCJ9fX0=
-      cached-skin-uuid-name: tektronic
+      cached-skin-uuid-name: andreiMCU
       player-skin-use-latest: false
-      cached-skin-uuid: 87fb3c6c-5ff6-4bf6-b326-992c6137ea12
-      player-skin-name: tektronic
+      cached-skin-uuid: 2db62a3560654c2c964defc5e0ace18a
+      player-skin-name: andreiMCU
```
Comment (malonnnn) [inline Citizens/templates.yml]: why is all this changed since you never changed the skin?

Comment (AnielMCU) [inline Citizens/templates.yml]: I'll do it

File: Citizens/templates.yml [hunk 2]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
-    navigator:
+      player-skin-signature:Nqw/8KSpjDasvNXh/4igYzRqCBo17ILVIu1n4El+tYdRGly31tWJ7qM7UwV5vvQeKwxIG0A4OtOQoBwkhYRXPlZxQI2zHqVIRSlNquiZBOik6ED8W0eu64DdYjZttaH7yAlbWsjMN4+MhHD761vk6R+Co50AmKJDN2yyEFKNyQFOIAPmTXlyr95IeLD4RvVhZ9QB1CszybUa6aRd0OpVZNDXSAqkHiLZULNeqGjYCI+wetT88GWk2S7hCEHRAZInwPBEs03OfkVa3MD+HfWCSs8SKVX8KhuW2+mxV/vDw12H1O/GTr1GgT9j+p7msLxuPiAkP1PsJcobrdrfb2vVLYYAI2DdLdYrIn5zA88nAik65FWQ80H2Y381sxqTHxwjIVrJzRr4TlFUi0YmtY//SkM+a3ST/4sCTWzxFO8DKXFWfHc+VwkK5NCtA/0xv5AmQPgjBlo4NtuCP7TLaGL3H2ehj7s8bS0qJS48mlC48rRDzChbM+jUjVs/SzMwcXFnzYe8Pw+FqlAJjjXXi0DpBCUFAlgBspPltfVq1Gp2kX8plz/Cg4hWUzzNdRDarJgTGjT815Eovajjf1OcehgHM2UCg1YJUapeCBuohtOmsdCbpInR1ZefrUaSgg3JAr3xNpAHpjV9qx5M+FZMZ0OvErzKuIO0Dw8hBY0ddlNja3E=     
+      navigator:
```
Comment (malonnnn) [inline Citizens/templates.yml]: remove 2 spaces

File: Citizens/templates.yml [hunk 3]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
-    navigator:
+      player-skin-signature:Nqw/8KSpjDasvNXh/4igYzRqCBo17ILVIu1n4El+tYdRGly31tWJ7qM7UwV5vvQeKwxIG0A4OtOQoBwkhYRXPlZxQI2zHqVIRSlNquiZBOik6ED8W0eu64DdYjZttaH7yAlbWsjMN4+MhHD761vk6R+Co50AmKJDN2yyEFKNyQFOIAPmTXlyr95IeLD4RvVhZ9QB1CszybUa6aRd0OpVZNDXSAqkHiLZULNeqGjYCI+wetT88GWk2S7hCEHRAZInwPBEs03OfkVa3MD+HfWCSs8SKVX8KhuW2+mxV/vDw12H1O/GTr1GgT9j+p7msLxuPiAkP1PsJcobrdrfb2vVLYYAI2DdLdYrIn5zA88nAik65FWQ80H2Y381sxqTHxwjIVrJzRr4TlFUi0YmtY//SkM+a3ST/4sCTWzxFO8DKXFWfHc+VwkK5NCtA/0xv5AmQPgjBlo4NtuCP7TLaGL3H2ehj7s8bS0qJS48mlC48rRDzChbM+jUjVs/SzMwcXFnzYe8Pw+FqlAJjjXXi0DpBCUFAlgBspPltfVq1Gp2kX8plz/Cg4hWUzzNdRDarJgTGjT815Eovajjf1OcehgHM2UCg1YJUapeCBuohtOmsdCbpInR1ZefrUaSgg3JAr3xNpAHpjV9qx5M+FZMZ0OvErzKuIO0Dw8hBY0ddlNja3E=
```
Comment (malonnnn) [inline Citizens/templates.yml]: needs a space after the :

File: Citizens/templates.yml [hunk 4]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
     metadata:
-      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0NzI3ODAzMDkyMzUsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiZDczOTcxMTY2MTM2YjEyNmZmN2U1Y2VmYzIyNTgzODYxYTczZjU3NjFiMTBmNTg4OGUzZDg5ZGE1ZTllMCJ9fX0=
-      cached-skin-uuid-name: tektronic
+      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0ODk5Njg1NTE5MzYsInByb2ZpbGVJZCI6IjJkYjYyYTM1NjA2NTRjMmM5NjRkZWZjNWUwYWNlMThhIiwicHJvZmlsZU5hbWUiOiJBbmRyZWlNQ1UiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzU5ZjNhZWNlNzUxNWQ1YjVhNzk4ODM1ODkzMWI2OGIyNmJhMjVlOTVlZTdhOWE0NWVmYzMzNzUyYTFmIn19fQ==
+      cached-skin-uuid-name: andreiMCU
       player-skin-use-latest: false
-      cached-skin-uuid: 87fb3c6c-5ff6-4bf6-b326-992c6137ea12
-      player-skin-name: tektronic
+      cached-skin-uuid: 2db62a3560654c2c964defc5e0ace18a
+      player-skin-name: andreiMCU
```
Comment (malonnnn) [inline Citizens/templates.yml]: no captial letters here

File: Citizens/templates.yml [hunk 5]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
         pitch: 0.0
     metadata:
-      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0NzI3ODAzMDkyMzUsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiZDczOTcxMTY2MTM2YjEyNmZmN2U1Y2VmYzIyNTgzODYxYTczZjU3NjFiMTBmNTg4OGUzZDg5ZGE1ZTllMCJ9fX0=
-      cached-skin-uuid-name: tektronic
+      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0ODk5Njg1NTE5MzYsInByb2ZpbGVJZCI6IjJkYjYyYTM1NjA2NTRjMmM5NjRkZWZjNWUwYWNlMThhIiwicHJvZmlsZU5hbWUiOiJBbmRyZWlNQ1UiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzU5ZjNhZWNlNzUxNWQ1YjVhNzk4ODM1ODkzMWI2OGIyNmJhMjVlOTVlZTdhOWE0NWVmYzMzNzUyYTFmIn19fQ==
+      cached-skin-uuid-name: andreiMCU
       player-skin-use-latest: false
-      cached-skin-uuid: 87fb3c6c-5ff6-4bf6-b326-992c6137ea12
-      player-skin-name: tektronic
+      cached-skin-uuid: 2db62a3560654c2c964defc5e0ace18a
```
Comment (malonnnn) [inline Citizens/templates.yml]: you need to use the UUID with dashes

File: Citizens/templates.yml [hunk 6]
```diff
@@ -8005,14 +8005,14 @@ atom:
         yaw: -90.0
         pitch: 0.0
     metadata:
-      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0NzI3ODAzMDkyMzUsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiZDczOTcxMTY2MTM2YjEyNmZmN2U1Y2VmYzIyNTgzODYxYTczZjU3NjFiMTBmNTg4OGUzZDg5ZGE1ZTllMCJ9fX0=
-      cached-skin-uuid-name: tektronic
+      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0ODk5Njg1NTE5MzYsInByb2ZpbGVJZCI6IjJkYjYyYTM1NjA2NTRjMmM5NjRkZWZjNWUwYWNlMThhIiwicHJvZmlsZU5hbWUiOiJBbmRyZWlNQ1UiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzU5ZjNhZWNlNzUxNWQ1YjVhNzk4ODM1ODkzMWI2OGIyNmJhMjVlOTVlZTdhOWE0NWVmYzMzNzUyYTFmIn19fQ==
+      cached-skin-uuid-name: andreiMCU
```
Comment (malonnnn) [inline Citizens/templates.yml]: no captial letters here

Comment (AnielMCU) [inline Citizens/templates.yml]: where?

File: Citizens/templates.yml [hunk 7]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
         pitch: 0.0
     metadata:
-      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0NzI3ODAzMDkyMzUsInByb2ZpbGVJZCI6Ijg3ZmIzYzZjNWZmNjRiZjZiMzI2OTkyYzYxMzdlYTEyIiwicHJvZmlsZU5hbWUiOiJUZWt0cm9uaWMiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2NiZDczOTcxMTY2MTM2YjEyNmZmN2U1Y2VmYzIyNTgzODYxYTczZjU3NjFiMTBmNTg4OGUzZDg5ZGE1ZTllMCJ9fX0=
-      cached-skin-uuid-name: tektronic
+      player-skin-textures: eyJ0aW1lc3RhbXAiOjE0ODk5Njg1NTE5MzYsInByb2ZpbGVJZCI6IjJkYjYyYTM1NjA2NTRjMmM5NjRkZWZjNWUwYWNlMThhIiwicHJvZmlsZU5hbWUiOiJBbmRyZWlNQ1UiLCJzaWduYXR1cmVSZXF1aXJlZCI6dHJ1ZSwidGV4dHVyZXMiOnsiU0tJTiI6eyJ1cmwiOiJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlLzU5ZjNhZWNlNzUxNWQ1YjVhNzk4ODM1ODkzMWI2OGIyNmJhMjVlOTVlZTdhOWE0NWVmYzMzNzUyYTFmIn19fQ==
+      cached-skin-uuid-name: andreimcu
       player-skin-use-latest: false
-      cached-skin-uuid: 87fb3c6c-5ff6-4bf6-b326-992c6137ea12
-      player-skin-name: tektronic
+      cached-skin-uuid: 2db62a3560654c2c964defc5e0ace18a
```
Comment (malonnnn) [inline Citizens/templates.yml]: you need to use the UUID with spaces

File: Citizens/templates.yml [hunk 8]
```diff
@@ -8005,14 +8005,14 @@ atom:
...
-    navigator:
+      player-skin-signature:Nqw/8KSpjDasvNXh/4igYzRqCBo17ILVIu1n4El+tYdRGly31tWJ7qM7UwV5vvQeKwxIG0A4OtOQoBwkhYRXPlZxQI2zHqVIRSlNquiZBOik6ED8W0eu64DdYjZttaH7yAlbWsjMN4+MhHD761vk6R+Co50AmKJDN2yyEFKNyQFOIAPmTXlyr95IeLD4RvVhZ9QB1CszybUa6aRd0OpVZNDXSAqkHiLZULNeqGjYCI+wetT88GWk2S7hCEHRAZInwPBEs03OfkVa3MD+HfWCSs8SKVX8KhuW2+mxV/vDw12H1O/GTr1GgT9j+p7msLxuPiAkP1PsJcobrdrfb2vVLYYAI2DdLdYrIn5zA88nAik65FWQ80H2Y381sxqTHxwjIVrJzRr4TlFUi0YmtY//SkM+a3ST/4sCTWzxFO8DKXFWfHc+VwkK5NCtA/0xv5AmQPgjBlo4NtuCP7TLaGL3H2ehj7s8bS0qJS48mlC48rRDzChbM+jUjVs/SzMwcXFnzYe8Pw+FqlAJjjXXi0DpBCUFAlgBspPltfVq1Gp2kX8plz/Cg4hWUzzNdRDarJgTGjT815Eovajjf1OcehgHM2UCg1YJUapeCBuohtOmsdCbpInR1ZefrUaSgg3JAr3xNpAHpjV9qx5M+FZMZ0OvErzKuIO0Dw8hBY0ddlNja3E=
```
Comment (malonnnn) [inline Citizens/templates.yml]: this needs a space after the :

File: Citizens/templates.yml [hunk 9]
```diff
@@ -887,7 +887,7 @@ captainatom:
       usedefaultstuckaction: true
     uuid: 1dc8679a-3a38-43e2-807a-985598c54a95
     traitnames: spawned,type,inventory,owner,gravity,location
-    name: Captain Atom
+    name: Captain
```
Comment (malonnnn) [inline Citizens/templates.yml]: put Captain Atom's name back

Comment (AnielMCU) [inline Citizens/templates.yml]: what ok

File: Citizens/templates.yml (hunk 2 above)
Comment (AnielMCU) [inline Citizens/templates.yml]: where?

File: Citizens/templates.yml [hunk 10]
```diff
@@ -887,7 +887,7 @@ captainatom:
       usedefaultstuckaction: true
     uuid: 1dc8679a-3a38-43e2-807a-985598c54a95
     traitnames: spawned,type,inventory,owner,gravity,location
-    name: Captain Atom
+    name: CaptainAtom
```
Comment (malonnnn) [inline Citizens/templates.yml]: forgot to add a space here

Comment (AnielMCU) [inline Citizens/templates.yml]: ok

**Pull request closed without merging**